content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -25,36 +20,19 @@ This guide assumes the following settings:
   export KF_DIR=<path to your Kubeflow application directory>
   ``` 
 
-* The `${CONFIG_FILE}` environment variable contains the path to your 
-  Kubeflow configuration file.
-
-  ```
-  export CONFIG_FILE=${KF_DIR}/{{% config-file-gcp-iap %}}
-  ```
-
-    Or:
+## Deleting your deployment
 
-  ```
-  export CONFIG_FILE=${KF_DIR}/{{% config-file-gcp-basic-auth %}}
-  ```
 
-For further background about the above settings, see the guide to
-[deploying Kubeflow with the CLI](/docs/gke/deploy/deploy-cli).
+1. To just delete the applications running in the Kubeflow namespace just delete that namespace
 
-## Deleting your deployment
+   ```
+   kubectl delete namespace kubeflow
+   ```
 
-Run the following commands to delete your deployment and reclaim all GCP
-resources:
+1. To delete the cluster and all GCP resources
 
 ```
-# If you want to delete all the resources, including storage:
-kfctl delete -f ${CONFIG_FILE} --delete_storage
-
-# If you want to preserve storage, which contains metadata and information
-# from Kubeflow Pipelines:
-kfctl delete -f ${CONFIG_FILE}
+cd ${KF_DIR}
+make delete-gcp
```

Comment:
Do we preserve the persistence disk?
I'd hope we reserve a command to keep it, that allows reinstallation without data loss.

Reply:
This will delete everything. If you don't want to delete the disk you would have to delete the other resources but not the disk.